### ISSUE TITLE
`isDataAvailable` can be run in parallel

### ISSUE THREAD

**Issue by terencechain** [prysmaticlabs/prysm#12843]
The current design assumes blobs mostly arrive after the block, but this claim has yet to be validated on testnets nor mainnet. 
If blobs arrive simultaneously as a block, there may be gains by executing `isDataAvailable` in parallel. From `isDataAvailable`, the bottlenecks are retrieving blobs from DB and running the verification proofs which probably takes 50ms (my rough guess)

**Comment by terencechain** [comment]
no longer relevant with the new blob sidecar format
